react/ContactsList/ContactRow.spec.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
+import ContactRow from './ContactRow'
+import renderer from 'react-test-renderer'
+describe('ContactRow', () => {
+  test('should accept the strict minimum', () => {
+    const contact = { email: [{ address: 'johndoe@localhost' }] }
+    const contactRowInstance = <ContactRow contact={contact} />
+    const contactrow = mount(contactRowInstance)
+    const contactrowemail = contactrow.find('ContactEmail')
+    expect(contactrowemail).toBeDefined()
```

Comment:
I think it is unnecessary to use toBeDefined() as text() will fail if called on 0 or more than 1 node.